src/controller.ts

```diff
@@ -42,7 +42,7 @@ export class AugurNodeController {
     this.running = true;
     this.errorCallback = errorCallback;
     try {
-      this.db = await createDbAndConnect(this.augur, this.networkConfig, this.databaseDir);
+      this.db = await createDbAndConnect(this.augur, this.networkConfig, errorCallback, this.databaseDir);
```

Comment:
Can you move the callback to the beginning for consistency?

Reply:
Or end?

Reply:
can not be at the end because of option databaseDir param.

seems augur is passed in as first param as general convention.

Reply:
Will move the error callback to the beginning